go/apps/mixnet/mixnet_test.go

```diff
@@ -84,6 +93,27 @@ func makeContext(batchSize int) (*RouterContext, *ProxyContext, error) {
 	return router, proxy, nil
 }
 
+func makeProxyContext(proxyAddr string) (*ProxyContext, error) {
+
+	// Create a domain with a LiberalGuard.
+	_, err := makeTrivialDomain(configDir)
+	if err != nil {
+		return nil, err
+	}
+
+	// CrateDomain() saves the configuration to disk; delete this now since
```

Comment:
Note that this is deferring deletion rather than deleting it now. Maybe drop the defer part?